src/main/java/io/strimzi/kafka/bridge/Application.java

```diff
@@ -62,12 +61,13 @@
     private static final int DEFAULT_EMBEDDED_HTTP_SERVER_PORT = 8080;
 
     @SuppressWarnings({"checkstyle:NPathComplexity"})
+    @SuppressFBWarnings("REC_CATCH_EXCEPTION")
```

Comment:
why this one?

Reply:
When I ran `mvn clean install -DskipTests spotbugs:check` it spit out this warning / error ...

Reply:
this is not the reason, this is the detection of the problem ... I would like to know why there is now a problem that needs this suppress warning that wasn't there before.

Reply:
No idea, I was just following spotbugs error report.
Perhaps checkstyle plugin upgrade?

Reply:
I would try to understand what's the error and then at least opening an issue with the reason to track and doing a change later to avoid doing it in the current PR. If spotbugs is raising this maybe we don't see a potential bug ... it's worth investigating.

Reply:
I think it's the issue of "eating" the exception:
* https://github.com/archived-codacy/codacy-findbugs/blob/master/src/main/resources/docs/description/REC_CATCH_EXCEPTION.md

Which in our case that's not the (completely the) case -- we log and exit the app ...

Reply:
The link you pasted seems to suggest that we should not catch just `Exception` as the main does but `RuntimeException` or being more specific. Would it be the solution instead of suppress the warning?

Reply:
Imho, we can leave it as is ... as that's from main() and we log and exit.